crossbar/bridge/rest/test/__init__.py

```diff
@@ -87,10 +85,10 @@ def _utcnow():
 
 def makeSignedArguments(params, signKey, signSecret, body):
 
-    params[b'timestamp'] = [network_string(_utcnow())]
+    params[b'timestamp'] = [bytes(_utcnow(), 'latin-1')]
```

Comment:
Reference on why that's using `latin-1` encoding: https://pythonhosted.org/six/#six.b

Reply:
Maybe make your comment above a comment in the code? (Because I can 100% assure you I'm going to ask that question again if I randomly look at this line ;) )

Reply:
(Oh, why isn't my other comment showing here? Anyway just in case: maybe turn the above in to a code-comment with the link please :) )

Reply:
cool, I removed the _utcnow() function and reused it from autobahn.